app/assets/javascripts/controllers/auth_key_pair_cloud/auth_key_pair_cloud_controller.js

```diff
@@ -1,15 +1,16 @@
 ManageIQ.angular.app.controller('keyPairCloudFormController', ['$http', '$scope', 'keyPairFormId', 'miqService', function($http, $scope, keyPairFormId, miqService) {
+    var vm = this;
     var init = function() {
-        $scope.keyPairModel = {
+        vm.keyPairModel = {
             name: '',
             public_key: '',
             ems_id: ''
         };
-        $scope.formId = keyPairFormId;
-        $scope.afterGet = false;
-        $scope.modelCopy = angular.copy( $scope.keyPairModel );
-        $scope.model = 'keyPairModel';
-        $scope.ems_choices = [];
+        vm.formId = keyPairFormId;
+        vm.afterGet = false;
+        vm.modelCopy = angular.copy( vm.keyPairModel );
+        vm.model = 'keyPairModel';
+        vm.ems_choices = [];
```

Comment:
Please add `vm.saveable = miqService.saveable;` so Add button knows when to enable itself.